clang/include/clang/CIR/Dialect/IR/CIRTypes.td

```diff
@@ -309,11 +316,15 @@ def CIR_FuncType : CIR_Type<"Func", "func"> {
     /// Returns the number of arguments to the function.
     unsigned getNumInputs() const { return getInputs().size(); }
 
+    /// Returns the result type of the function as an actual return type or
+    /// explicit !cir.void
```

Comment:
Same comments here.

Reply:
In this case, "explicit" is correct.  The definition of `getReturnType` returns a `!cir.void` if the MLIR type has no return type.  This function should be called when looking at things from a C++ point of view instead of an MLIR point of view.